cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,108 @@
+                    return {'error': 'Strava credentials not found in Secret Manager'}, 400, cors_headers
+                raise
+        else:
+            return {'error': 'Bad Request - Invalid data_source'}, 400, cors_headers
+
+    except auth.InvalidIdTokenError:
+        return {'error': 'Unauthorized - Invalid token'}, 401, cors_headers
+    except auth.ExpiredIdTokenError:
+        return {'error': 'Unauthorized - Token expired'}, 401, cors_headers
+    except auth.RevokedIdTokenError:
+        return {'error': 'Unauthorized - Token revoked'}, 401, cors_headers
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)